node/engine/engine.go

```diff
@@ -145,14 +143,14 @@ func New(vm *payments.VoucherManager, msg messageservice.MessageService, chain c
 
 	e.eventHandler = eventHandler
 
-	logging.ConfigureZeroLogger()
-	e.logger = logging.WithAddress(zerolog.New(logDestination).With().Timestamp(), e.store.GetAddress()).Caller().Logger()
+	// We set a local logger here that will include the address of the node when logging
+	e.logger = slog.Default().With(logging.WithAddressAttribute(*e.store.GetAddress()))
 
 	e.policymaker = policymaker
 
 	e.vm = vm
 
-	e.logger.Print("Constructed Engine")
+	e.logger.Info("Constructed Engine")
```

Comment:
The only reason we have a `logger` on the engine is to reduce some boilerplate code to add the store address when logging. This statement would also work if we wrote it like 
```
slog.Info("Constructed Engine", logging.WithAddressAttribute(*e.store.GetAddress()))
```